frontend/lib/laletterbuilder/letter-builder/routes.tsx

```diff
@@ -12,6 +15,21 @@ import { LALetterBuilderOnboardingStep } from "./step-decorators";
 import { LALetterBuilderWelcome } from "./welcome";
 
 const LALetterBuilderAskName = LALetterBuilderOnboardingStep(AskNameStep);
+const LALetterBuilderAskCityState = LALetterBuilderOnboardingStep((props) => (
+  <AskCityState
+    {...props}
+    confirmModalRoute={LALetterBuilderRoutes.locale.letter.cityConfirmModal}
+  >
+    <p>must be California</p>
```

Comment:
Just flagging that this is not translated, but this is placeholder text I assume so ok.